### ISSUE TITLE
Automatically import types from '@polkadot/types/interfaces/runtime' in typegen scripts

### ISSUE THREAD

**Issue by herryho** [polkadot-js/api#3149]
Hi, 

When I use typegen to generate types, if I use the types from '@polkadot/types/interfaces/runtime', for example Permill, Balance, AssetId, AccountId, BlockNumber, etc., the script cannot recongnize these types, and I have to mannually import the types into the types.ts files.
![image](https://user-images.githubusercontent.com/45537372/107007576-adfe1a80-67cd-11eb-8c9b-89a897643f4c.png)

Is it possible to automatically detect whether the types, if not provided by customization, exist in the  '@polkadot/types/interfaces/runtime' and automatically import them as how it does to the primitive types such as u32?

Thanks!

**Comment by jacogr** [comment]
It does import it automatically - the API itself uses the typegen scripts for interface generation, see for instance the output and the auto-use from runtime here https://github.com/polkadot-js/api/blob/master/packages/api/src/augment/consts.ts#L6

And here are examples from others users with published types (i.e. users of the scripts) -

https://github.com/hicommonwealth/edgeware-node-types/blob/master/src/interfaces/augment-api-consts.ts#L7
https://github.com/hicommonwealth/edgeware-node-types/blob/master/src/interfaces/augment-api-query.ts#L22

**Comment by herryho** [comment]
Thank Jaco. I also used Edgeware as my template to generate my types, but strangely the only missing type imports are from '@polkadot/types/interfaces/runtime'. 

After reading the links you provided above, I finally found out my problem. I created a type folder named "runtime" under "packages/types/src/interfaces", the same structure as the polkadot api types. That might be what confused the typegen scripts. 

Now everything is fine after the runtime folder is deleted.
